api/integreatly/v1alpha1/grafanadashboard_types.go

```diff
@@ -32,15 +32,17 @@ import (
 
 // GrafanaDashboardSpec defines the desired state of GrafanaDashboard
 type GrafanaDashboardSpec struct {
-	Json             string                            `json:"json,omitempty"`
-	Jsonnet          string                            `json:"jsonnet,omitempty"`
-	Plugins          PluginList                        `json:"plugins,omitempty"`
-	Url              string                            `json:"url,omitempty"`
-	ConfigMapRef     *corev1.ConfigMapKeySelector      `json:"configMapRef,omitempty"`
-	Datasources      []GrafanaDashboardDatasource      `json:"datasources,omitempty"`
-	CustomFolderName string                            `json:"customFolderName,omitempty"`
-	GrafanaCom       *GrafanaDashboardGrafanaComSource `json:"grafanaCom,omitempty"`
+	Json                 string                            `json:"json,omitempty"`
+	Jsonnet              string                            `json:"jsonnet,omitempty"`
+	Plugins              PluginList                        `json:"plugins,omitempty"`
+	Url                  string                            `json:"url,omitempty"`
+	ConfigMapRef         *corev1.ConfigMapKeySelector      `json:"configMapRef,omitempty"`
+	Datasources          []GrafanaDashboardDatasource      `json:"datasources,omitempty"`
+	CustomFolderName     string                            `json:"customFolderName,omitempty"`
+	GrafanaCom           *GrafanaDashboardGrafanaComSource `json:"grafanaCom,omitempty"`
+	ContentCacheDuration *metav1.Duration                  `json:"contentCacheDuration,omitempty"`
```

Comment:
We haven't been the best at doing this as you can see but lets start working on it. Please describe what ContentCacheDuration does.

Just put a comment or two above the definition that you create. For example:
```.go
// ContentCacheDuration sets how often the operator should resync with the grafana API.
// This to make sure that dashboard and the CRD looks the same at all time.
```